### PR TITLE
Platform admin can approve vendor

### PR DESCRIPTION
**Pull request by chadellison**

### CODE REVIEW

File: app/controllers/pending_controller.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+class PendingController < ApplicationController
+  def index
```
Comment (Jbern16) [inline app/controllers/pending_controller.rb]: hmm could this method just be a method in application controller? And then call @vendors?

**Comment by Jbern16** [comment]
hey chad, this looks good. I am just a little confused on the need for the pending controller

File: app/controllers/vendors_controller.rb [hunk 2]
```diff
@@ -5,6 +5,29 @@ def show
   end
 
   def index
-    @vendors = Vendor.where(status: "Active")
+    @vendors = Vendor.where(status: 1)
```
Comment (s-espinosa) [inline app/controllers/vendors_controller.rb]: Psyched that this is an enum.

File: app/helpers/platform_admin_vendors_helper.rb [hunk 3]
```diff
@@ -0,0 +1,9 @@
+module PlatformAdminVendorsHelper
+  def pending_vendors(vendors)
+    vendors.where(status: "pending")
+  end
```
Comment (s-espinosa) [inline app/helpers/platform_admin_vendors_helper.rb]: Can't we keep this in the Vendors model? Something like:

def self.pending
  where(status: "pending")
end

def self.inactive
  where(status: "inactive")
end

Then we could call it from the views using @vendors.pending and @vendors.inactive. We could have a similar method for @vendors.active

File: app/controllers/pending_controller.rb (hunk 1 above)
Comment (s-espinosa) [inline app/controllers/pending_controller.rb]: Didn't we agree to remove this controller?

**Comment by chadellison** [comment]
I think the convention here is to have it in a helper because it is for presenting in the view. I won't be a stickler about it though if you would like me to put it in the model.

**Pull request merged**